redbot/cogs/audio/core/commands/controller.py

```diff
@@ -306,9 +315,12 @@ async def command_seek(self, ctx: commands.Context, seconds: Union[int, str]):
 
         Accepts seconds or a value formatted like 00:00:00 (`hh:mm:ss`) or 00:00 (`mm:ss`).
         """
-        dj_enabled = self._dj_status_cache.setdefault(
-            ctx.guild.id, await self.config.guild(ctx.guild).dj_enabled()
-        )
+        if ctx.guild.id in self._dj_status_cache:
+            dj_enabled = self._dj_status_cache[ctx.guild.id]
+        else:
+            dj_enabled = self._dj_status_cache.setdefault(
+                ctx.guild.id, await self.config.guild(ctx.guild).dj_enabled()
+            )
         vote_enabled = await self.config.guild(ctx.guild).vote_enabled()
```

Comment:
I'm pretty sure you wanted to switch to call to `all()` here:
```suggestion
        guild_data = await self.config.guild(ctx.guild).all()
        dj_enabled = self._dj_status_cache.setdefault(ctx.guild.id, guild_data["dj_enabled"])
        vote_enabled = guild_data["vote_enabled"]
```